src/EFCore/ModelBuilder.cs

```diff
@@ -250,6 +338,23 @@ public virtual ModelBuilder Ignore([NotNull] Type type)
             return this;
         }
 
+        /// <summary>
+        ///     Excludes the given entity type from the model. This method is typically used to remove types from
```

Comment:
From the description I wasn't sure what would happen if the given name was not in the model or represented something other than an entity type. I suggest the below, but feel free to rephrase if my assumptions are not correct.

```suggestion
        ///     Excludes an entity type with the given name from the model if it exists. This method is typically used to remove types from
```